### PR TITLE
Remove nullable disable in the checks.

### PR DESCRIPTION
**Pull request by andy840119**
Remove nullable disable annotation in the files belongs to the namespace `osu.Game.Rulesets.Edit.Checks` and `osu.Game.Tests.Editing.Checks`.
~~Will open another PR for the rulesets.~~

And sorry for pause contribution for a while because busy recently ;_;

### CODE REVIEW

File: osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs [hunk 1]
```diff
@@ -128,13 +126,13 @@ private BeatmapVerifierContext getContext(Texture background, [CanBeNull] Stream
...
         /// <param name="stream">The stream representing the background file.</param>
-        private Mock<IWorkingBeatmap> getMockWorkingBeatmap(Texture background, [CanBeNull] Stream stream = null)
+        private Mock<IWorkingBeatmap> getMockWorkingBeatmap(Texture? background, Stream? stream = null)
         {
             stream ??= new MemoryStream(new byte[1024 * 1024]);
 
             var mock = new Mock<IWorkingBeatmap>();
             mock.SetupGet(w => w.Beatmap).Returns(beatmap);
-            mock.SetupGet(w => w.Background).Returns(background);
+            mock.SetupGet(w => w.Background).Returns(background.AsNonNull());
```
Comment (andy840119) [inline osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs:135]: Guess `AsNonNull()` can be removed again if remove the nullable annotation in the `IWorkingBeatmap` and mark Background property as nullable.

File: osu.Game.Tests/Editing/Checks/CheckAudioQualityTest.cs [hunk 2]
```diff
@@ -43,7 +42,7 @@ public void TestMissing()
 
             var mock = new Mock<IWorkingBeatmap>();
             mock.SetupGet(w => w.Beatmap).Returns(beatmap);
-            mock.SetupGet(w => w.Track).Returns((Track)null);
+            mock.SetupGet(w => w.Track).Returns(default(Track?).AsNonNull());
```
Comment (andy840119) [inline osu.Game.Tests/Editing/Checks/CheckAudioQualityTest.cs:45]: Not really sure it's the right usage of `AsNonNull()` but i think it should be better than `(Track)null!` 🤔

Comment (peppy) [reply osu.Game.Tests/Editing/Checks/CheckAudioQualityTest.cs:45]: This definitely looks wrong.

File: osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs (hunk 1 above)
Comment (peppy) [inline osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs:135]: Just make the parameter non-nullable.

Comment (andy840119) [reply osu.Game.Tests/Editing/Checks/CheckBackgroundQualityTest.cs]: fixed 🙆🏼‍♂️

File: osu.Game.Tests/Editing/Checks/CheckZeroByteFilesTest.cs [hunk 3]
```diff
@@ -74,7 +73,7 @@ private BeatmapVerifierContext getContext(long byteLength)
         private BeatmapVerifierContext getContextMissing()
         {
             var mockWorkingBeatmap = new Mock<IWorkingBeatmap>();
-            mockWorkingBeatmap.Setup(w => w.GetStream(It.IsAny<string>())).Returns((Stream)null);
+            mockWorkingBeatmap.Setup(w => w.GetStream(It.IsAny<string>())).Returns(default(Stream?).AsNonNull());
```
[the remark this answers is not in the document]
Comment (peppy) [reply osu.Game.Tests/Editing/Checks/CheckZeroByteFilesTest.cs:76]: Can this be `(Stream)null!`?

[the remark this answers is not in the document]
Comment (andy840119) [reply osu.Game.Tests/Editing/Checks/CheckZeroByteFilesTest.cs]: Sorry not notice that.
Fixed 🙆🏼‍♂️

**Review by peppy — APPROVED** [review]
Seems okay to get these in.

**Pull request merged**